docs/README.md

```diff
@@ -582,6 +582,8 @@ Users can set the following custom annotations:
 
 * `topic` - the topic annotation is used with the event-connector pattern, if at least one event-connector is installed on the OFC installation.
 
+* `schedule` - the schedule annotation is used with the cron-connector.
```

Comment:
I think a link to the cron-connector (either the OF docs, or the project repo) should be included here so that someone reading this, who may not be aware of the project, can see what a "cron-connector" is, and how to use it.

Also, perhaps adjust the language a little to indicate that the cron-connector is a function, again to avoid any confusion. 

`the schedule annotation is used with the [cron-connector](link) function`

Reply:
Done.